src/components/textarea/textarea.component.tsx

```diff
@@ -59,20 +147,21 @@ const Textarea = ({
   "data-role": dataRole,
   helpAriaLabel,
   ...props
-}) => {
+}: TextareaProps) => {
   const { validationRedesignOptIn } = useContext(NewValidationContext);
-  const computeLabelPropValues = (prop) =>
+  const computeLabelPropValues = <T,>(prop: T): undefined | T =>
```

Comment:
```suggestion
  const computeLabelPropValues = <T>(prop: T): undefined | T =>
```

I haven't tried it but is there any reason we need the comma there? It looks weird to me.

Reply:
I too this type definition from @mkrds' code in the [Textbox component](https://github.com/Sage/carbon/blob/FE-4839-convert-textarea-to-typescript/src/components/textbox/textbox.component.tsx#L186). It seems to require the comma.